calblueprint/unloop
# ViewMoreModal and ActionItemModal work for ActionItems

implemented ViewMoreModal (generalized the CaseNoteCardModal) and the ActionItemModal now works for editing action items on the action item creation page. The final button layout might cause merge conflicts with Calvin's PR since his button layout will be different and conflict. AddToTemplates functionality doesn't work yet but can be added later/during hackathon (would be an easy addition since it's just an API call).

## Note:
I also had to tweak some of the styling for ActionItemCard because it wasn't rendering properly. Make sure that it renders nicely on your end when you pull before merging into master.

## To test:
Pull and edit action items that were both created from templates and from scratch and make sure nothing funky happens. When you edit an action item created from a template the checkmark goes away. If that's not what you want I can also change that.


### Screenshots
View modal
![image](https://user-images.githubusercontent.com/35501399/80873520-049e5d00-8c6e-11ea-9e67-4a361b69393c.png)

Edit modal
![image](https://user-images.githubusercontent.com/35501399/80873524-1122b580-8c6e-11ea-97a1-56a2e96a39af.png)

What the cards should look like
![image](https://user-images.githubusercontent.com/35501399/80873527-17189680-8c6e-11ea-8b30-a37f2d858ed6.png)



CC: @didvi
